styles/main/style.css

```diff
@@ -403,18 +404,69 @@ html {
   justify-content: center;
 }
 
+/* Achievements section */
+.achievements {
+  padding: 200px 0px;
+  position: relative;
+  z-index: 0;
+}
+.achievements__box {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  gap: 160px;
+}
+.achievements__box:before {
+  content: "Project";
+  position: absolute;
+  top: 20%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: var(--achievements-background-gray);
+  font-size: 300px;
+  letter-spacing: 0.13em;
+  text-align: center;
+  z-index: -1;
+  max-width: 100%;
+  line-height: 1;
+}
+.achievement__holder {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+}
```

Comment:
Actually the way this should be styled is by adding a max width to each achievement and then centering it inside that container. Just like in the design. If you look closely there is no gap between the items, they just take all of the width.

![image](https://user-images.githubusercontent.com/68795596/210561736-7990cfa9-22ff-4375-9402-34ffe2e10fe0.png)